RevokeCgn/function.json

```diff
@@ -0,0 +1,25 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "cgn/revoke/{fiscalCode}",
+      "methods": [
+        "post"
+      ]
```

Comment:
About API design: what about `DELETE /cgn/{fiscalCode}`?

Reply:
Since `CANCELED` status change can be triggered by a scheduled function, maybe we can use DELETE method don't we ? @gunzip

Reply:
yup